distributed/distributed.yaml

```diff
@@ -17,7 +17,8 @@ distributed:
     # after they have been removed from the scheduler
     events-cleanup-delay: 1h
     idle-timeout: null      # Shut down after this duration, like "1h" or "30 minutes"
-    transition-log-length: 100000
+    transition-log-length: 5000
```

Comment:
If we want to change tihs default value then let's do so in another PR.  I recommend reverting this for now.

Reply:
fine for me 👍